components/gitpod-db/src/typeorm/team-db-impl.ts

```diff
@@ -186,20 +186,15 @@ export class TeamDBImpl extends TransactionalDBImpl<TeamDB> implements TeamDB {
             throw new ResponseError(ErrorCodes.BAD_REQUEST, "Please choose a name that is at most 64 characters long.");
         }
 
-        // Storing new entry in a TX to avoid potential dupes caused by racing requests.
-        const em = await this.getEntityManager();
```

Comment:
Need to remove the migration from here, as we are already wrapping it with one. The `gitpod-service` is the only call-site that I could find.

Reply:
Can't we nest transactions if we use the same entity manager?